data/json/items/gunmod/underbarrel.json

```diff
@@ -437,6 +437,33 @@
     "description": "A minimalist pump-action shotgun, which can be mounted under the barrel of many rifles.  It allows a total of four shotgun shells to be loaded and fired.  This one was modified and customized to mount on pretty much any weapon other than pistols and pump-action guns, if you so want.",
     "mod_targets": [ "smg", "rifle", "shotgun", "launcher", "crossbow" ]
   },
+  {
+    "id": "m26_mass",
+    "type": "GUNMOD",
+    "name": { "str": "M26-MASS shotgun" },
+    "description": "Modular Accessory Shotgun System, an underbarrel bolt-action shotgun, inspired by \"Masterkey\", and fixed most of his problems.  It can be mounted under the barrel of different rifles, or used as standalone weapon by attachment to a pistol grip or collapsible buttstock module.  Uses 3 or 5 shell magazines",
```

Comment:
```suggestion
    "description": "Modular Accessory Shotgun System, an underbarrel bolt-action shotgun, inspired by \"Masterkey\", and fixed most of its problems.  It can be mounted under the barrel of different rifles, or used as standalone weapon by attachment to a pistol grip or collapsible buttstock module.  It feeds from three or five shot detachable magazines.",
```